Userland/Applications/Spreadsheet/Cell.cpp

```diff
@@ -191,19 +191,21 @@ void Cell::copy_from(const Cell& other)
     m_thrown_value = other.m_thrown_value;
 }
 
-CellUndoCommand::CellUndoCommand(Cell& cell)
+CellUndoCommand::CellUndoCommand(Cell& cell, String& previous_data)
```

Comment:
```suggestion
CellUndoCommand::CellUndoCommand(Cell& cell, String const& previous_data)
```

Reply:
Thanks for the feedback! I can agree. My development process isn't really pertinent to this feature.

... as for the `const` addition, I'll add that :-) +1 for const-correctness!